src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1279,10 +1280,9 @@ bool layout_optimizer::are_layouts_suitable_for_onednn(program_node& node) {
 }
 
 bool layout_optimizer::is_primitive_implemented_for_onednn(program_node& node) {
-    if (node.is_type<fully_connected>() || node.is_type<gemm>() || node.is_type<pooling>() ||
-        node.is_type<convolution>() || node.is_type<deconvolution>() ||
-        node.is_type<reduce>() || node.is_type<reorder>() || node.is_type<concatenation>()) {
-            return true;
+    if (node.type()->does_possible_implementation_exist(node, *(node.get_kernel_impl_params()), cldnn::impl_types::onednn) ||
+        node.is_type<deconvolution>()) {
```

Comment:
why is it necessary to handle deconv separately?

Reply:
deconv has onednn impl. But does_possbile_implementation_exist returns true for checkking onednn impl of deconv. I just added condition for now. I will check root cause